ngi_pipeline/conductor/launchers.py

```diff
@@ -20,7 +20,15 @@ def get_engine_for_bp(project, config=None, config_file_path=None):
     :param NGIProject project: The project to get the engine from.
     """
     charon_session = CharonSession()
-    best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
+    try:
+        best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
```

Comment:
Yeah I view Charon with the same level of distrust I view the User. Sometimes he has the right values when I go looking for them and sometimes he doesn't, and other times he claims that he's never even heard of such a key.
In this case it is sometimes the value is blank and recently this threw a KeyError, so better safe than sorry